tests/hindi.txt~

```diff
@@ -0,0 +1,20 @@
+कांग्रेस पार्टी अध्यक्ष सोनिया गांधी, तमिलनाडु की मुख्यमंत्री जयललिता और रिज़र्व बैंक के गवर्नर रघुराम राजन के बीच एक समानता है.
```

Comment:
I guess this is a save file created by your editor and should not be included in the repository